index.js

```diff
@@ -0,0 +1,97 @@
+let clickHandler = function(event){
+    if(event.target.parentNode.className == "done"){
+        event.target.src = event.target.src.endsWith("unticked.png")? "ticked.jpeg" : "unticked.png";
```

Comment:
instead of checking the image itself, you can save an attribute called `data-checked` (for instance), and pass it true or false.
You can see documentation about this here:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes